### PR TITLE
test: selenium test setup

### PR DESCRIPTION
**Pull request by Fluctuationqt**

### CODE REVIEW

File: modules/integration-tests/pom.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    <dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+      <version>${selenium.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.github.bonigarcia</groupId>
+      <artifactId>webdrivermanager</artifactId>
+      <version>${selenium.webdriver.manager.version}</version>
```
Comment (ThuF) [inline modules/integration-tests/pom.xml:98]: Test scope?

File: modules/integration-tests/pom.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  <properties>
+    <license.header.location>../../licensing-header.txt</license.header.location>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+      <version>${selenium.version}</version>
```
Comment (ThuF) [inline modules/integration-tests/pom.xml:92]: Test scope?

File: modules/integration-tests/pom.xml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            <version>2.22.2</version>
+            <configuration>
+              <excludes>
+                <exclude>**/*ITest.java</exclude>
+              </excludes>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
```
Comment (ThuF) [inline modules/integration-tests/pom.xml:47]: Do we need this profile? Isn't the `integration-tests` profile enough?

**Review by ThuF — CHANGES_REQUESTED** [review]
Few minor comments

**Review by ThuF — APPROVED** [review]
+1

**Comment by vmutafov** [comment]
Related to: https://github.com/SAP/xsk/issues/382

**Pull request merged**